### PR TITLE
Feat smartmeter set tariff on meter

### PR DESCRIPTION
**Pull request by hannek24**
connects to osgp/platform#243

### CODE REVIEW

**Comment by hannek24** [comment]
https://github.com/OSGP/Platform/issues/243

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java [hunk 1]
```diff
@@ -81,12 +81,20 @@ public DayProfile getSunday() {
         return this.sunday;
     }
 
+    // @formatter:off
```
Comment (rudidevries) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java]: You should not commit commented out code.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java [hunk 2]
```diff
@@ -81,12 +81,20 @@ public DayProfile getSunday() {
...
+                ", \n\t\t\t\t\t monday=" + this.monday +
+                ", \n\t\t\t\t\t tuesday=" + this.tuesday +
+                ", \n\t\t\t\t\t wednesday=" + this.wednesday +
+                ", \n\t\t\t\t\t thursday=" + this.thursday +
+                ", \n\t\t\t\t\t friday=" + this.friday +
+                ", \n\t\t\t\t\t saturday=" + this.saturday +
+                ", \n\t\t\t\t\t sunday=" + this.sunday +
+                "\n\t\t\t\t]";
     }
+    // @formatter:on
```
Comment (rudidevries) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java]: You should not commit commented out code.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/DayProfileAction.java [hunk 3]
```diff
@@ -28,7 +28,7 @@ public Integer getScriptSelector() {
     }
 
     public Date getStartTime() {
-        return new Date(this.startTime.getTime());
+        return this.startTime;
```
Comment (rudidevries) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/DayProfileAction.java]: This way the startTime property is mutable. Looking at the diff, it was correct before..

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java (hunk 1 above)
Comment (hannek24) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java]: This is need so the code is not (wrongly) formatted.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java (hunk 2 above)
Comment (hannek24) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/WeekProfile.java]: Needed so the code is not formatted during save.

File: osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/DayProfileAction.java (hunk 3 above)
Comment (hannek24) [inline osgp-dto/src/main/java/com/alliander/osgp/dto/valueobjects/smartmetering/DayProfileAction.java]: Fixed

**Pull request merged**